### DISCUSSION TITLE
Welcome to vega-app Discussions!

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nagato58** [comment]
can someone help me download this

**Comment by sagarh1** [comment]
Dear dev,
I request you please build this app for the ios users..

**Comment by rahulgupta1432** [comment]
Just wanted to know
Will you provide us backend repository of Vega app?
